src/Infrastructure/Array/interface/ESMF_ArrayGet.cppF90

```diff
@@ -129,7 +129,7 @@ contains
 ! !INTERFACE:
   ! Private name; call using ESMF_ArrayGet()
   subroutine ESMF_ArrayGetDefault(array, keywordEnforcer, arrayspec, typekind, &
-    rank, localarrayList, indexflag, distgridToArrayMap, &
+    rank, replicatedDimCount, localarrayList, indexflag, distgridToArrayMap, &
```

Comment:
Does this seem like a good place to insert this new optional output argument? Another possible position is after `dimCount`, though the stuff there seems related solely to the `distGrid`, so that didn't feel quite as appropriate.